scripts-dev/build_debian_packages.py

```diff
@@ -23,7 +23,6 @@
 # These are expanded inside the dockerfile to be a fully qualified image name.
 # e.g. docker.io/library/debian:bullseye
 DISTS = (
-    "debian:buster",  # oldstable: EOL 2022-08
```

Comment:
I've made a follow-up PR to document the distribution EOL date and EOL date forced by the Python version that each distribtion uses -> https://github.com/matrix-org/synapse/pull/15909

This way the connection is more obvious and hopefully we can catch this next time we want to remove a Python version.